docs/.vuepress/config.js

```diff
@@ -21,7 +21,20 @@ export default defineUserConfig({
 
   theme: defaultTheme({
     logo: "https://vuejs.press/images/hero.png",
-    navbar: ["/databases/", "/workflows/", "/dashboard-components/", "/cloud-functions/", "/internationalization/", "/integrating-new-apps/"],
+    navbar: [
+      {
+        text: "Documentation",
+        children: [
+          { text: "Databases", link: "/databases/" },
+          { text: "Workflows", link: "/workflows/" },
+          { text: "Dashboard Components", link: "/dashboard-components/" },
+          { text: "Cloud Functions", link: "/cloud-functions/" },
+          { text: "Internationalization", link: "/internationalization/" },
+          { text: "Integrating New Apps", link: "/integrating-new-apps/" },
+          { text: "Assets Optimization", link: "/assets-optimization/" },
```

Comment:
Could I suggest that the header be rephrased to `Optimizing Assets` to match the tone of the previous header (`Integrating New Apps`).

Reply:
Of course ! Thank you for the sugestion :)